frontend/src/components/Inquiries/AddInquiry.tsx

```diff
@@ -12,41 +11,16 @@ interface AddInquiryProps {
 }
 
 const AddInquiry = ({ isOpen, onClose }: AddInquiryProps) => {
-  const fields: FieldDefinition<InquiryCreate>[] = [
-    {
-      name: "text",
-      label: "Inquiry Text",
-      placeholder: "Enter the text of your inquiry.",
-      type: "textarea",
-      validation: {
-        required: "Inquiry text is required.",
-        minLength: {
-          value: MIN_INQUIRY_LENGTH,
-          message: `Inquiry must be at least ${MIN_INQUIRY_LENGTH} characters.`,
-        },
-        maxLength: {
-          value: MAX_INQUIRY_LENGTH,
-          message: `Inquiry can not be greater than ${MAX_INQUIRY_LENGTH} characters.`,
-        },
-        validate: (value: string) =>
-          isValidUnicode(value) || "Inquiry must be a valid unicode string.",
-      },
-      inputProps: {
-        "data-testid": "add-inquiry-text",
-      },
-    },
-  ]
-
   const mutationFn = async (data: InquiryCreate): Promise<void> => {
     await InquiriesService.inquiriesCreateInquiry({ requestBody: data })
   }
-
+  const content = "Why do birds suddenly appear every time you are near?"
   return (
     <FormModal<InquiryCreate>
       isOpen={isOpen}
       onClose={onClose}
-      title="Add Inquiry"
-      fields={fields}
+      title="Why birds?"
+      content={content}
       mutationFn={mutationFn}
```

Comment:
Here we just use the content input rather than the form fields.  

Would we ever want to do both?  Probably?

Reply:
I considered a similar approach. When you click `Submit` its going to try to call the mutationFn with the data passed in from the form.  But since we aren't using the form, you may end up passing additional props and adding more conditional logic to the modal. Its certainly do-able but I was hesitant to go down that route given that we are at the end the program.

Reply:
I absolutely appreciate the question of where we leave things at the end of the program.  i defer to others on that.